### ISSUE TITLE
Improving the application graphical design.

### ISSUE THREAD

**Issue by pparent76** [nanu-c/axolotl#778]
HI,

Just want to mention that I'm starting to use the app for a daily usage and one of the things that lowers user-friendlyness and pleasure to use. I think it could be easily improved taking inspiration on Teleport, Android signal app, or whatsapp. I think it is within my skill and therefore I'm thinking about proposing and improval when I have time (maybe not before few weeks), if you're ok with it.

For example the first and biggest issue I see is that the font for messages is way to big, and therefore we can only read few words on screen. It is way bigger than on any other messenging app on ubports.

Also I think overall the colors could be better chosen, maybe reuse the colors of the original app. Maybe have the contact logo have different colors depending on the name, to avoid being too monotonous. Maybe have a special color, or visual thing for groups so that we can easily differentiate them visually. Among other things.

Thank you.

**Comment by Flaburgan** [comment]
Hello Pierre and thank you for your interest about Axolotl.
Indeed, the UI can be greatly improved, if you feel like you want to contribute then you're very welcome! Don't hesitate if you have any questions.

**Comment by pparent76** [comment]
Ok well as soon as I have time (maybe in few weeks) I will try to go over the most obvious improvements at least for my own usage, and I will publish my modifications here in github, so you can see if you like it.

The only possibly tedious part for me is to setup a cross-compilation environment to compile ubports package for my vollaphone. But I hope it won't be too tedious.

I'll keep you up.

**Comment by Flaburgan** [comment]
No need to work on your phone, you can work locally on desktop ;)

**Comment by Flaburgan** [comment]
Actually, the first thing you can do to help would be to tackle https://github.com/nanu-c/axolotl/issues/185 which is about inserting default data in the local DB so that you can then work on your computer without needing a phone number etc.

**Comment by pparent76** [comment]
Yes but I prefer to cross-compile it for my phone, It's better to have a good overview of the actual Ux on a phone, and on top of that it gives direct reward for developing: being able to actually improve the look&feel for my own usage.

**Comment by pparent76** [comment]
Ok I'm going to try working on it today.

**Comment by pparent76** [comment]
I someone can help me with building the package for uBports it might help.

**Comment by Flaburgan** [comment]
It is describe in the doc: https://github.com/nanu-c/axolotl/blob/main/docs/INSTALL.md#clickable if you have an actual issue please describe it
